doc/modules/mod_muc_light.md

```diff
@@ -109,6 +109,14 @@ Specifies maximal number of rooms returned for a single Disco request.
 
 When enabled, rooms the user occupies are included in their roster.
 
+### `modules.mod_muc_light.allow_multiple_admin`
+* **Syntax:** boolean
+* **Default:** `true`
```

Comment:
Not sure but I suppose the default value should be `false`. Those code changes should not affect any previous work or behavior.